targets/gnome

```diff
@@ -8,9 +8,15 @@ HOSTBIN='startgnome'
 CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 . "${TARGETSDIR:="$PWD"}/common"
 
+if release -ge wheezy -ge kali -ge trusty; then
```

Comment:
+1 for `release -lt jessie -lt sana -lt trusty` and swap flashback/fallback below.